part-customization.tex

```diff
@@ -50,18 +50,77 @@ \section{Introduction}
 \section{Workflow configuration}
 \label{sec-customization-workflow-configuration}
 
-LedgerSMB uses the concept of workflows\index{workflows} to manage the document life cycle.
-An invoice for example may be \texttt{saved}, \texttt{posted} or \texttt{reversed}.  These
-are states in the life cycle of the invoice.  Each state may have associated actions.  In
-case of the invoice an example may be to \texttt{Post} it when it's \texttt{saved} or to
-\texttt{E-mail} it when it's \texttt{posted}.
+LedgerSMB uses the concept of workflows\index{workflows} to manage object life cycle.
+An invoice, for example, may be \texttt{saved}, \texttt{posted} or \texttt{reversed}.  These
+are states in the life cycle of the invoice.  Each state may have associated actions. E.g. in
+case of the invoice there may be a \texttt{Post} action when it's \texttt{saved} or to
+\texttt{E-mail} it when it's \texttt{posted}.  Actions may cause the document to change state:
+if the \texttt{Post} action is executed on a \texttt{saved} invoice, its new state
+will be \texttt{posted}.  E-mailing an invoice does not change its state: it
+remains posted.  Instead, a new workflow is created which manages state of the e-mail.
+
+\digraph[scale=0.4]{wf1}{

```

Comment:
This digraph needs to be labeled as a Figure.

Reply:
Done.